orbitdb.go

```diff
@@ -87,6 +90,14 @@ func (n *NewOrbitDBOptions) applyDefaults() {
 		n.Logger = zap.NewNop()
 	}
 	n.Logger = n.Logger.Named("odb")
+
+	if n.GroupMetadataStoreType == "" {
+		n.GroupMetadataStoreType = "berty_group_metadata"
```

Comment:
```suggestion
		n.GroupMetadataStoreType = "weshnet_group_metadata"
```

Reply:
I think it should be generic and each client will set its own store type.

Reply:
Interesting. So we will update the Berty code to set "berty_group_metadata" and "berty_group_messages" ?

Reply:
> Interesting. So we will update the Berty code to set "berty_group_metadata" and "berty_group_messages" ?

Yes, it's better to do it now because it's fresh in our minds.